### PR TITLE
add test vector for shared-stated retry service, see #114

### PR DESCRIPTION
**Pull request by william-zk**
see issue: #114

### CODE REVIEW

File: draft-ietf-quic-load-balancers.md [hunk 1]
```diff
@@ -1797,6 +1797,35 @@ cid 10b3f367d8627b36990a28d67f50b97846 sid 5e018f0197 su 2289cae06a566e5cb6cfa4
 cid 1024412bfe25f4547510204bdda6143814 sid 8a8dd3d036 su 4b12933a135e5eaaebc6fd
 ~~~
 
+## Share-state Retry Token
```
Comment (martinduke) [inline draft-ietf-quic-load-balancers.md:1800]:
```suggestion
## Shared-state Retry Token
```

File: draft-ietf-quic-load-balancers.md [hunk 2]
```diff
@@ -1797,6 +1797,35 @@ cid 10b3f367d8627b36990a28d67f50b97846 sid 5e018f0197 su 2289cae06a566e5cb6cfa4
...
+Shared-State Retry Service Token:
+unique_token_number 0xcc9d8ecc83cfba681ec5d0ac
+key_sequence 0x00
+encrypted_shared_state_retry_service_token_body 0x20b73beca4e84ff23cc8931d5dc152
+83d712b5e5e96907cca56323936887db6ee3c0656e1254bb9930e69b
+AEAD_ICV 0x0b4a256b79ff8df4df4ef054481e2da9
+
+AEAD related parameters:
+AEAD_nonce 0xfdafbdf8b6f98d5027f5e19e
+AEAD_associated_data 0x0100007f000000000000000000000000cc9d8ecc83cfba681ec5d0ac00
```
Comment (martinduke) [inline draft-ietf-quic-load-balancers.md:1826]: The IP address is backwards. It should appear as it does it does in the IP header: 7f000001.

**Review by martinduke — CHANGES_REQUESTED** [review]
When I fake my inputs to use the inverted IP address as you have here, I get the same output. So I think your code is sound except for that.

**Comment by william-zk** [comment]
> When I fake my inputs to use the inverted IP address as you have here, I get the same output. So I think your code is sound except for that.

Sounds great, I have rewrite the test-vector and give more information about it,please confirm again if you are free

**Review by martinduke — APPROVED** [review]
Checks out, thanks

**Pull request merged**